package.json

```diff
@@ -22,7 +25,7 @@
   "scripts": {
     "lint": "eslint && prettier --check .",
     "lint:fix": "eslint --fix && prettier --write .",
-    "test": "jest",
+    "test": "NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" jest",
```

Comment:
The Snapshot-test code for Flat Config is written as ES Modules, and in order to have Jest run ES Modules, you need to write it like this.

cf. [ECMAScript Modules · Jest](https://jestjs.io/docs/ecmascript-modules)